lib/grr.js

```diff
@@ -1399,13 +1403,244 @@ function stepListCRs(arg, cb) {
+                            arg.reviewArg);
+                        // prompt for user input, and apply CR
+                        var prompt_schema = {
+                            properties: {
+                                answer: {
+                                    description: review_desc,
+                                    pattern: /[y|n]/,
+                                    message: 'y or n will do!',
+                                    required: true
+                                }
+                            }
+                        };
+                        prompt.colors = false;
+                        prompt.message = '';
+                        prompt.start();
+                        prompt.get(
+                            prompt_schema,
+                            function user_input(prompt_err, result) {
+                                if (prompt_err) {
+                                    next(new VError(
+                                        prompt_err,
+                                        'problem trying to prompt user'));
+                                    return;
+                                }
+                                if (result.answer == 'y') {
+                                    console.log(extsprintf(
+                                        'Granting a \'%s\' +1 to this review',
+                                        arg.reviewArg));
+                                    common.forkExecWaitAndLog({
+                                        argv: ['ssh', '-q',
+                                            '-o', 'StrictHostKeyChecking=no',
+                                            '-o', 'UserKnownHostsFile=/dev/null',
+                                            '-p', '29418',
+                                            arg.gerritUsername + '@cr.joyent.us',
+                                            'gerrit', 'review',
+                                            '--project', cr_res.project,
+                                            '--label', arg.reviewArg + '=+1',
+                                            cr_res.currentPatchSet.revision
+                                        ],
+                                        log: arg.log
+                                    // eslint-disable-next-line no-unused-vars
+                                    }, function (fork_err, review_res) {
+                                        if (fork_err) {
+                                            next(fork_err);
+                                            return;
+                                        }
+                                    });
+                                } else {
+                                    console.log('skipping this review.');
+                                }
+                                next();
+                        });
+                    } catch (parseErr) {
```

Comment:
What's this parseErr meant to be around? Can it be more tightly around the part that can throw?

Reply:
Revisiting this, I can't tell exactly what we were trying to do :-/ Removing the try/catch and testing didn't yield anything, so I'll take it out.

My best guess is that this was possibly for the 'var gerrit_res = JSON.parse(element);' in stepQueryCRs() potentially blowing up, but that being the case, that's where we should have this try/catch block. I'll do that there, and for stepListCRs() while I'm at it.